apis/v1alpha1/adminnetworkpolicy_types.go

```diff
@@ -129,6 +129,7 @@ type AdminNetworkPolicyIngressRule struct {
 	//
 	// Support: Core
 	//
+	// +kubebuilder:validation:Enum={"Allow", "Deny", "Pass"}
```

Comment:
I think you can just put this validation once on the `AdminNetworkPolicyRuleAction` type!

Reply:
ack trying out:
```
// +enum
// +kubebuilder:validation:Enum={"Allow", "Deny", "Pass"}
type AdminNetworkPolicyRuleAction string
```
instead.

Reply:
ah yeah that works too, in the end the yamls generated are the same

Reply:
updating this to make it centralized